larray/core/session.py

```diff
@@ -908,43 +915,55 @@ def summary(self, template=None):
         return '\n'.join(template.format(**kwargs) for kwargs in templ_kwargs)
 
 
-def local_arrays(depth=0):
+def _exclude_private_array(vars_dict):
```

Comment:
s/_exclude_private_array/_exclude_private_vars/ ? (this is not specific to arrays)